_delphi_utils_python/delphi_utils/weekday.py

```diff
@@ -44,7 +46,7 @@ def get_params(data, denominator_col, numerator_cols, date_col, scales, logger):
         return params
 
     @staticmethod
-    def _fit(X, scales, npnums, npdenoms):
+    def _fit(X, scales, npnums, npdenoms, solver=cp.CLARABEL):
```

Comment:
dont specify a default here; this is only called by `get_params()` which should be explicit about the solver.
```suggestion
    def _fit(X, scales, npnums, npdenoms, solver):
```